### ISSUE TITLE
Freezing parameters instead of the full layer

### ISSUE THREAD

**Issue by aegroto** [LaurentMazare/tch-rs#551]
Hello,

What's the best way using this wrapper to stop tracking the gradient of some specific parameters of a layer? As of now I have been able to freeze a layer as a whole. Thanks in advance!

**Comment by LaurentMazare** [comment]
Maybe you can try using `set_requires_grad(false)` on the parameters of the layer? E.g. for a conv2d layer that would be `conv2d.bs.set_requires_grad(false)` to freeze the biases only. Fwiw this is what gets used when freezing a `VarStore` [here](https://github.com/LaurentMazare/tch-rs/blob/fbe70bb1460ccddb73f04695e05d4758c9728557/src/nn/var_store.rs#L255).

**Comment by aegroto** [comment]
Thanks for your quick response @LaurentMazare. So I guess that if I want to freeze only part of a Tensor I have to undo the optimization step right after the update instead of stop tracking the gradient.
